spec/graphql/queries/dogs_spec.rb

```diff
@@ -5,18 +5,25 @@
     user1 = create(:user)
```

Comment:
This is probably going to conflict with what I did this afternoon, I'll try to get it pulled down and resolved tonight.